### DISCUSSION TITLE
YouTubeChannel2RssFeed does not exist in configuration

### DISCUSSION THREAD

**Discussion by roughnecks** [FreshRSS/FreshRSS#6630] [Q&A]
Hello, I opened an issue here: https://github.com/cn-tools/cntools_FreshRssExtensions/issues/26
not understanding what the message is telling me, and got no reply so far.. maybe anyone here knows?

Thanks

**Comment by Alkarex** [comment]
Check your FreshRSS version and extension version and make sure they are both at the latest release

**Comment by roughnecks** [comment]
They are.

Il 17 luglio 2024 17:38:12 CEST, Alexandre Alapetite ***@***.***> ha scritto:

**Comment by roughnecks** [comment]
![image](https://github.com/user-attachments/assets/9de4416f-e243-4a8c-927d-269afb1ad964)

![image](https://github.com/user-attachments/assets/cb7291f6-d8ac-423e-b4a7-2041a17be11f)
